server/container/guice/guice-common/src/main/java/org/apache/james/jmap/JMAPModule.java

```diff
@@ -106,6 +106,8 @@ public void initModule() {
                     "MOVE support in MailboxManager is required by JMAP Module");
             Preconditions.checkArgument(mailboxManager.getSupportedMessageCapabilities().contains(MailboxManager.MessageCapabilities.Attachment),
                     "Attachment support in MailboxManager is required by JMAP Module");
+            Preconditions.checkArgument(mailboxManager.getSupportedSearchCapabilities().contains(MailboxManager.SearchCapabilities.MultimailboxSearch),
```

Comment:
We thought about that with matthieu, and decided to put it in mailboxManager for the same reason than message capabilities are in mailboxManager instead of messageManager: MessageSearchIndex like MessageManager need a MailboxManager.